### ISSUE TITLE
PHPStorm @noinspection causes Missing Import warning

### ISSUE THREAD

**Issue by renjinsk** [Haehnchen/idea-php-annotation-plugin#190]
While working with PHPStorm and adding a `@noninspection` annotation for a file or method phpdoc block it will cause an annotation warning with the message `[Annotations] Missing import`
To "silence" it you have to add another `noinspection` annotation `AnnotationMissingUseInspection`

So, in case you do this for the file, you endup with something like this
```php
<?php
/** @noinspection AnnotationMissingUseInspection */
/** @noinspection PhpUndefinedClassInspection */
```
This happens with the latest version [7.0.0](https://github.com/Haehnchen/idea-php-annotation-plugin/releases/tag/7.0.0)

**Comment by blowski** [comment]
Related - I'm getting "Missing import" warnings for PHPUnit annotations, like `@test`

**Comment by Haehnchen** [comment]
fixed with `7.0.1`

**Comment by blowski** [comment]
That was incredibly quick, thanks @Haehnchen
